core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/ValidateRequest.java

```diff
@@ -254,11 +264,19 @@ void setRuleParameters(Map<String, Object> ruleParameters) {
 	private static final class IssuesToPersist {
 		
 		public final String ruleId;
-		public final Collection<ComponentIdentifier> affectedComponentIds;
-
-		public IssuesToPersist(String ruleId, Collection<ComponentIdentifier> affectedComponentIds) {
+		public Collection<ValidationIssueDetails> issueDetails;
+		
+		@SuppressWarnings("unchecked")
+		public <T> IssuesToPersist(String ruleId, Collection<T> evaluationResult) {
```

Comment:
This `<T>` type parameter is completely unnecessary. Just use the `?` wildcard type instead.